### PR TITLE
workaround to publish also .gitignore

### PR DESCRIPTION
**Pull request by sebastianovide**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   },
   "scripts": {
     "start": "HTTPS=true react-scripts start",
-    "build": "react-scripts build && cp -a ./build/. ./cordova-app/www/",
+    "build": "react-scripts build && cp -a ./build/. ./cordova-app/www/ && cp .gitignore gitignore",
```
Comment (joykuotw) [inline package.json]: Why need to copy to a new gitignore folder?

Comment (sebastianovide) [reply package.json]: it is not a folder: it is a copy of .gitignore. it is a work around as .gitignore is not published.

File: bin/geovation-photos-update.js [hunk 2]
```diff
@@ -39,7 +39,8 @@ function copySrc() {
   ncp(path.join(geovationPhotoFolder, "src"), "src", errFunc);
   ncp(path.join(geovationPhotoFolder, "cordova-app"), "cordova-app", errFunc);
   ncp(path.join(geovationPhotoFolder, "public"), "public", errFunc);
-  ncp(path.join(geovationPhotoFolder, ".gitignore"), ".gitignore", { clobber : false }, errFunc);
+  // see https://github.com/atlassubbed/atlas-npm-init/issues/1
+  ncp(path.join(geovationPhotoFolder, "gitignore"), ".gitignore", { clobber : false }, errFunc);
```
Comment (joykuotw) [inline bin/geovation-photos-update.js]: Whey set `clobber` to false? it means `ncp` will not overwrite the existing `.gitignore` file.

Comment (sebastianovide) [reply bin/geovation-photos-update.js]: correct... (should we instead. ?.. perhaps using it we'll learn)

Comment (joykuotw) [reply bin/geovation-photos-update.js]: sure, we can leave it for now. I will merge this PR then.

**Pull request merged**